system/Bootstrap.cfc

```diff
@@ -546,6 +551,32 @@ component {
 		return ExpandPath( "/preside" );
 	}
 
+	private string function _getAppBasePath( string appBasePath ) {
+		var appBasePath = "";
+
+		if ( Len( arguments.appBasePath ) ) {
+			appBasePath = arguments.appBasePath;
+		} else {
+			var cbController = _getColdboxController();
```

Comment:
@DominicWatson 
Trying to get this to work based on site.path if no configuration options are passed through / set in the applications config.
Thought I had this working locally but just realised I was passing the site path in the applications app cfc file.
cbController is returning as null when cfg options are removed.